src/main/java/pageobject_model/test/WebDriverSeleniumGridTest.java

```diff
@@ -14,27 +16,41 @@
 import java.net.URL;
 import java.time.Duration;
 
-import static org.testng.AssertJUnit.assertTrue;
+import static org.testng.AssertJUnit.assertEquals;
+
 
 public class WebDriverSeleniumGridTest {
+    private static final String GRID_HOST = "http://localhost:4444/wd/hub";
 
-    @Test
-    public void seleniumGridTest() throws URISyntaxException, IOException {
+    private WebDriver driver;
+
+    private CloudGoogleComputeEnginePage computeEnginePage;
+
+    @BeforeMethod(alwaysRun = true)
+    public void browserSetup() throws IOException, URISyntaxException {
```

Comment:
please move current tests to selenium Grid

Reply:
added to @BeforeMethod driver creation steps using RemoteWebDriver and DesiredCapabilities